### ISSUE TITLE
Error when precompiling

### ISSUE THREAD

**Issue by IljaK91** [queryverse/ElectronDisplay.jl#79]
I get the following error:
```
ERROR: LoadError: MethodError: no method matching join(::FilePathsBase.WindowsPath, ::String, ::String, ::String, ::String)
Closest candidates are:
  join(::IO, ::Any, ::Any, ::Any) at strings/io.jl:274
  join(::Any, ::Any, ::Any) at strings/io.jl:302
  join(::IO, ::Any, ::Any) at strings/io.jl:290
  ...
Stacktrace:
 [1] top-level scope at C:\Users\Ilja\.julia\packages\ElectronDisplay\J4pc3\src\ElectronDisplay.jl:12
 [2] include(::Function, ::Module, ::String) at .\Base.jl:380
 [3] include(::Module, ::String) at .\Base.jl:368
 [4] top-level scope at none:2
 [5] eval at .\boot.jl:331 [inlined]
 [6] eval(::Expr) at .\client.jl:467
 [7] top-level scope at .\none:3
in expression starting at C:\Users\Ilja\.julia\packages\ElectronDisplay\J4pc3\src\ElectronDisplay.jl:12
```

Some additional Info
```
(@v1.5) pkg> status
Status `C:\Users\Ilja\.julia\environments\v1.5\Project.toml`
  [70b36510] AutomaticDocstrings v1.0.1
  [6e4b80f9] BenchmarkTools v0.5.0
  [336ed68f] CSV v0.7.7
  [5d742f6a] CSVFiles v1.0.0
  [1375f997] CompEcon v0.4.0
  [8f4d0f93] Conda v1.4.1
  [a93c6f00] DataFrames v0.21.7
  [31a5f54b] Debugger v0.6.5
  [31c24e10] Distributions v0.23.8
  [634d3b9d] DrWatson v1.15.1
  [d872a56f] ElectronDisplay v1.0.0
  [442a2c76] FastGaussQuadrature v0.4.2
  [f6369f11] ForwardDiff v0.10.12
  [38e38edf] GLM v1.3.10
  [28b8d3ca] GR v0.51.0
  [c91e804a] Gadfly v1.3.0
  [19dc6840] HCubature v1.4.0
  [7073ff75] IJulia v1.21.3
  [a98d9a8b] Interpolations v0.12.10
  [aa1ae85d] JuliaInterpreter v0.7.26
  [b964fa9f] LaTeXStrings v1.1.0
  [2774e3e8] NLsolve v4.4.1
  [429524aa] Optim v0.22.0
  [9b87118b] PackageCompiler v1.2.1
  [d96e819e] Parameters v0.12.1
  [f0f68f2c] PlotlyJS v0.14.0
  [91a5bcdd] Plots v1.6.0
  [438e738f] PyCall v1.91.4
  [d330b81b] PyPlot v2.9.0
  [1fd47b50] QuadGK v2.4.1
  [fcd29c91] QuantEcon v0.16.2
  [1a8c2f83] Query v1.0.0
  [295af30f] Revise v2.7.4
  [f2b01f46] Roots v1.0.5
  [efcf1570] Setfield v0.7.0
  [90137ffa] StaticArrays v0.12.4
  [2913bbd2] StatsBase v0.33.1
  [4c63d2b9] StatsFuns v0.9.5
  [3eaba693] StatsModels v0.6.13
  [f3b207a7] StatsPlots v0.14.11
  [e88e6eb3] Zygote v0.5.5
  [c4c688b2] jlpkg v1.2.2
  [37e2e46d] LinearAlgebra
  [9a3f8284] Random
  [10745b16] Statistics
  [8dfed614] Test
```

```
julia> versioninfo()
Julia Version 1.5.1
Commit 697e782ab8 (2020-08-25 20:08 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i5-5200U CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, broadwell)
Environment:
  JULIA_EDITOR = "C:\Users\Ilja\AppData\Local\Programs\Microsoft VS Code\Code.exe"
  JULIA_NUM_THREADS = 4
```

**Comment by ericphanson** [comment]
I just ran into this too. It looks like the issue is https://github.com/rofinn/FilePathsBase.jl/pull/75, which was a breaking change but ElectronDisplay.jl wasn't updated to handle it. Julia's [compat] mechanism should handle that, but while FilePathsBase got a breaking release, FilePaths itself only had a patch release. Since ElectronDisplay.jl depends on FilePaths but not FilePathsBase, the transient dependency got updated resulting in this issue.

I guess the fix is for ElectronDisplay to depend on FilePathsBase and explicitly import its join method to use here? cc @rofinn
